python_examples/sim_speeds/simulation_outputs/output.csv

```diff
@@ -0,0 +1,62 @@
+time,InfectionStatus.Susceptible,InfectionStatus.Exposed,InfectionStatus.InfectASympt,InfectionStatus.InfectMild,InfectionStatus.InfectGP,InfectionStatus.InfectHosp,InfectionStatus.InfectICU,InfectionStatus.InfectICURecov,InfectionStatus.Recovered,InfectionStatus.Dead

```

Comment:
Do we actually need to save outputs here - surely only the speed matters?

Reply:
Yes. In future we are likely to want other information to plot such as peak infected numbers

Reply:
I'm still not convinced by this - the population sizes we have below don't seem to be that hard to replicate? And given we haven't saved the parameters that we ran this with, it would likely be safer to replicate them in the future anyway